docs/proposals/job.md

```diff
@@ -0,0 +1,135 @@
+# Job Controller
+
+## Abstract
+A proposal for implementing a new controller - Job controller, which will be responsible
+for managing pod(s) that require to run on demand.
```

Comment:
I would add a little more detail as to why you need something to manage pods. I think if someone reads this abstract with a basic knowledge of Kubernetes, they would probably think that we already have a way to run pods on demand - just create them using `kubectl create`.